src/test/java/org/beanpod/switchboard/fixture/UserFixture.java

```diff
@@ -8,22 +8,21 @@
 public class UserFixture {
 
   public static SwitchBoardUserDetails getUserDetails() {
-    return new SwitchBoardUserDetails()
-        .builder()
-        .username("moh@gmail.com")
-        .password("1234.")
-        .build();
+    new SwitchBoardUserDetails();
+    return SwitchBoardUserDetails.builder().username("moh@gmail.com").password("1234.").build();
```

Comment:
Why `new SwitchboardUserDetails()`? Seems like it creates an object and does nothing with it. Same for the other 2 fixtures.

Reply:
Oups, I'll clean up the useless object instantiations.